addons/bus/tools/bus.py

```diff
@@ -0,0 +1,46 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from functools import wraps
+
+from odoo import models
+from odoo.http import request
+from odoo.addons.bus.websocket import wsrequest
+
+
+def notify_bus_on_completion(func):
+    @wraps(func)
+    def wrapper(self, *args, **kwargs):
+        bus_rpc_uuid = kwargs.pop("bus_rpc_uuid", None)
```

Comment:
There's a mismatch between this function, trying to grab the `bus_rpc_uuid` parameter from the keyword arguments and the client that sends it in a parameter called `context`.